project/PACKAGERS.md

```diff
@@ -123,40 +77,17 @@ from the upstream Golang perspective.
 
 ## Building Docker
 
-Please use our build script ("./hack/make.sh") for all your compilation of
-Docker. If there's something you need that it isn't doing, or something it could
-be doing to make your life as a packager easier, please get in touch with Tianon
-and help us rectify the situation. Chances are good that other packagers have
-probably run into the same problems and a fix might already be in the works, but
-none of us will know for sure unless you harass Tianon about it. :)
-
-All the commands listed within this section should be run with the Docker source
-checkout as the current working directory.
-
-### `AUTO_GOPATH`
-
-If you'd rather not be bothered with the hassles that setting up `GOPATH`
-appropriately can be, and prefer to just get a "build that works", you should
-add something similar to this to whatever script or process you're using to
-build Docker:
-
-```bash
-export AUTO_GOPATH=1
-```
-
-This will cause the build scripts to set up a reasonable `GOPATH` that
-automatically and properly includes both docker/docker from the local
-directory, and the local "./vendor" directory as necessary.
+Please use our build script ("./hack/make.sh") for compilation.
 
 ### `DOCKER_BUILDTAGS`
 
-If you're building a binary that may need to be used on platforms that include
+If you're building a binary that might be used on platforms that include
 AppArmor, you will need to set `DOCKER_BUILDTAGS` as follows:
 ```bash
 export DOCKER_BUILDTAGS='apparmor'
 ```
```

Comment:
Oh, I don't think this is actually used anymore?  Can someone else confirm too? :innocent:

Reply:
Indeed, looks like https://github.com/moby/moby/pull/42254 was the PR which finally removed our last `+build apparmor`.